tests/evidence.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for Turbinia evidence."""
+
+import json
+import unittest
+
+from turbinia import evidence
+from turbinia import TurbiniaException
+
+
+class TestTurbiniaEvidence(unittest.TestCase):
+  """Test evidence module."""
+
+  def testEvidenceSerialization(self):
+    """Test that evidence serializes/unserializes."""
+    e = evidence.RawDisk(
```

Comment:
Yeah, in general I don't use one-char var names either, but I guess in tests I am sometimes a bit more lax.  I'll fix these up, :)